.gitignore

```diff
@@ -29,3 +29,4 @@ juju-backup-*.tar.gz
 scripts/mgo-run-txn/mgo-run-txn
 tests/tmp.*
 *.out
+scripts/schemadocs/output.html
```

Comment:
This should be removed

Reply:
Pushed https://github.com/juju/juju/pull/11699/commits/51a6a9a49083ff230c79e0d3da9a73e8154f5e67 which removes this.